firefox-ios/Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewController.swift

```diff
@@ -222,11 +222,11 @@ class CreditCardBottomSheetViewController: UIViewController,
         let estimatedContentHeight = cardTableView.contentSize.height +
         buttonsHeight + UX.bottomSpacing + UX.distanceBetweenHeaderAndTop
         let aspectRatio = estimatedContentHeight / contentView.bounds.size.height
-        contentViewHeightConstraint.constant = contentViewHeightConstraint.constant * aspectRatio
+        contentViewHeightConstraint?.constant = (contentViewHeightConstraint?.constant ?? 0.0) * aspectRatio
```

Comment:
I am not sure if it is actually meaningful to set this to zero if it is null. In theory if it is null then you aren't even setting it so why does it matter? But, I think if you never expect it to be zero falling back to zero is a little weird? what about unwrapping it and only updating it if it is not nil?

Reply:
Nice catch! Totally agree 👍 

I also threw in another adjustment for something similar to this done in the init block (see latest commit) while I was at it, let me know if that's alright too.